exercises/high-scores/canonical-data.json

```diff
@@ -0,0 +1,91 @@
+{
+  "exercise": "high-scores",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Returns the list of scores",
+      "property": "scores",
+      "input": {
+        "scores": [3, 5, 2, 7]
+      },
+      "expected": [3, 5, 2, 7]
+    },
+    {
+      "description": "Returns the last score",
+      "property": "latest",
+      "input": {
+        "scores": [10, 0, 9, 3]
+      },
+      "expected": 3
+    },
+    {
+      "description": "Returns the highest score",
+      "property": "highest",
+      "input": {
+        "scores": [4, 10, 7]
+      },
+      "expected": 10
+    },
+    {
+      "description": "Returns the top 3 scores",
+      "cases": [
+        {
+          "description": "Returns personal bests",
+          "property": "top",
+          "input": {
+            "scores": [5, 3, 1]
+          },
+          "expected": [5, 3, 1]
+        },
+        {
+          "description": "Returns personal bests highest to lowest",
+          "property": "top",
+          "input": {
+            "scores": [2, 1, 3]
+          },
+          "expected": [3, 2, 1]
+        },
+        {
+          "description": "Returns personal bests when there are less than 3",
+          "property": "top",
+          "input": {
+            "scores": [3, 7]
+          },
+          "expected": [7, 3]
+        },
+        {
+          "description": "Returns personal bests when there is only one",
+          "property": "top",
+          "input": {
+            "scores": [4]
+          },
+          "expected": [4]
+        },
+        {
+          "description": "Returns three top scores from a long list",
+          "property": "top",
+          "input": {
+            "scores": [1, 3, 9, 3, 10, 2, 1, 0, 3, 4, 4, 7, 7]
+          },
+          "expected": [10, 9, 7]
+        }
+      ]
+    },
+    {
+      "description": "Returns the goal when highest is latest",
+      "property": "goal",
+      "input": {
+        "scores": [2, 4, 0, 3, 7]
+      },
+      "expected": "Your latest score was 7. That's your personal best!"
+    },
+    {
+      "description": "Returns the goal when not highest score",
+      "property": "goal",
+      "input": {
+        "scores": [2, 10, 0, 3, 7]
+      },
+      "expected": "Your latest score was 7. That's 3 short of your personal best!"
+    }
```

Comment:
Might be nice to make a distinction between a new personal best and a tied personal best.

Reply:
We're going to add a new test to specify what happens in case of a tie :+1: We don't want to add too many cases here so it's just going to return the same as the personal best.

Reply:
I've pushed the new test. Check "Returns report when there is a tie", please.